packages/tx/src/fromRpc.ts

```diff
@@ -28,7 +28,7 @@ export const normalizeTxParams = (_txParams: any): TxData => {
   txParams.r = txParams.r === '0x0' ? '0x' : txParams.r
   txParams.s = txParams.s === '0x0' ? '0x' : txParams.s
 
-  if (txParams.v !== '0x') {
+  if (txParams.v !== '0x' || txParams.r !== '0x' || txParams.s !== '0x') {
```

Comment:
Since I am now an Optimism expert (lol): Optimism system transactions (aka "deposited transactions", I would very much say these are synonyms, might need some confirmation though) have their own tx type (`0x7E` respectively `126`), so might be the cleaner option here to do this check based on a tx type check.